src/main/java/me/ryanhamshire/GriefPrevention/DatabaseDataStore.java

```diff
@@ -339,14 +340,18 @@ else if (this.getSchemaVersion() < 1)
                 List<String> containerNames = Arrays.asList(containersString.split(";"));
                 containerNames = this.convertNameListToUUIDList(containerNames);
 
+                String creaturesString = results.getString("creatures");
+                List<String> creaturesNames = Arrays.asList(creaturesString.split(";"));
+                creaturesNames = this.convertNameListToUUIDList(creaturesNames);
```

Comment:
Name list should already be a UUID list - this conversion is for legacy data. Doesn't need to be done.